test/EVMHost.cpp

```diff
@@ -132,6 +134,8 @@ EVMHost::EVMHost(langutil::EVMVersion _evmVersion, evmc::VM& _vm):
 	tx_context.tx_origin = 0x9292929292929292929292929292929292929292_address;
 	// Mainnet according to EIP-155
 	tx_context.chain_id = evmc::uint256be{1};
+	// The minimum value of basefee
+	tx_context.block_base_fee = evmc::bytes32{7};
```

Comment:
Ok, so we missed this in #11706.

Reply:
Though it also makes logically sense in this PR, so fine either way.

Reply:
Why 7?

Reply:
Ah that's the cost, nevermind

Reply:
For future reference: the minimum of 7 wei is not specified in the EIP. Here's some explanation on why it's 7 wei from ETH R&D discord: https://discord.com/channels/595666850260713488/749160974506000444/861666788495851561